packages/frontend/src/components/message/Message.tsx

```diff
@@ -860,16 +852,10 @@ function WebxdcMessageContent({ message }: { message: T.Message }) {
         {truncateText(info.name, 42)}
       </div>
       <div>{info.summary}</div>
-      {info.internetAccess && (
-        <div className='experimental'>
-          <b>EXPERIMENTAL</b> Webxdc that has full internet access, be careful!
-          (only works in saved messages)
-        </div>
-      )}
```

Comment:
So, is `internetAccess` only true for the map webxdc now? Do we never want this warning?

Reply:
It is set to true by Core if a webxdc is an "integrated" webxdc app (like map). Other apps don't have internet_access any more.